sdk/formrecognizer/azure-ai-formrecognizer/tests/conftest.py

```diff
@@ -69,4 +69,11 @@ def wrapper(*args, **kwargs):
         except HttpResponseError as error:
             if "Invalid request".casefold() in error.message.casefold():
                 pytest.mark.skip("flaky service response")
+            elif "Generic error".casefold() in error.message.casefold():
```

Comment:
Do you think we should include the flaky response in the skip message or log the response? Just so we don't miss somewhere where we skip a test that is failing due to something on our end.

Reply:
Done, added the message to the skip and logged since I'm still experimenting with what will be easier to see and analyze in devops